freqtrade/exchange/__init__.py

```diff
@@ -183,11 +184,34 @@ def get_ticker(pair: str, refresh: Optional[bool] = True) -> dict:
 def get_ticker_history(pair: str, tick_interval) -> List[Dict]:
     # TODO: tickers need to be in format 1m,5m
     # fetch_ohlcv returns an [[datetime,o,h,l,c,v]]
-    if not _API.markets:
-        _API.load_markets()
-    ohlcv = _API.fetch_ohlcv(pair, str(tick_interval)+'m')
+    if 'fetchOHLCV' not in _API.has or not _API.has['fetchOHLCV']:
+        raise OperationalException(
```

Comment:
should the `get_ticker_history` raise anything? I think it either gets the ticker history or then returns empty result